### PR TITLE
Soap Receiver sample and Soap producer client added. File Receiver sample updated

### PR DESCRIPTION
**Pull request by keizer619**

### CODE REVIEW

File: modules/samples/producers/soap-client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <parent>
+        <artifactId>cep-samples-producers</artifactId>
+        <groupId>org.wso2.cep</groupId>
+        <version>4.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>soap-client</artifactId>
+    <name>WSO2 CEP - Publisher - HTTP Client</name>
+
```
Comment (mohanvive) [inline modules/samples/producers/soap-client/pom.xml]: Please change the name to Soap Client

File: modules/samples/producers/soap-client/pom.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.analytics-common</groupId>
+            <artifactId>org.wso2.carbon.databridge.commons</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.analytics-common</groupId>
+            <artifactId>org.wso2.carbon.databridge.commons.thrift</artifactId>
+        </dependency>
+        <dependency>
```
Comment (mohanvive) [inline modules/samples/producers/soap-client/pom.xml]: Are sure that we need databridge dependencies here ?  Can you please add only necessary dependencies..

File: modules/samples/producers/soap-client/src/main/java/org/wso2/carbon/sample/soap/Soap.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+                } catch (XMLStreamException e) {
+                    e.printStackTrace();
+                } catch (AxisFault axisFault) {
+                    axisFault.printStackTrace();
+                }
+            }
+        } catch (Throwable t) {
+            t.printStackTrace();
+        }
```
Comment (mohanvive) [inline modules/samples/producers/soap-client/src/main/java/org/wso2/carbon/sample/soap/Soap.java]: Please log the exception.. Don't use printStackTrace(). If u found any other place fix them as wel..

File: modules/samples/producers/soap-client/pom.xml (hunk 1 above)
Comment (keizer619) [inline modules/samples/producers/soap-client/pom.xml]: Fixed

File: modules/samples/producers/soap-client/src/main/java/org/wso2/carbon/sample/soap/Soap.java (hunk 3 above)
Comment (keizer619) [inline modules/samples/producers/soap-client/src/main/java/org/wso2/carbon/sample/soap/Soap.java]: Fixed

File: modules/samples/producers/soap-client/pom.xml (hunk 2 above)
Comment (keizer619) [inline modules/samples/producers/soap-client/pom.xml]: i checked and we need data bridge agent to manage soap messages and endpoints. We can remove thrift and common. Fixed.

**Pull request merged**